third_party/move/move-compiler-v2/tests/reference-safety/bug_13952.exp

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
There must be a merge conflict, this file has been renamed, but is now added again with old name. Did you rebase?